docs/source/previous-versions.mdx

```diff
@@ -26,7 +26,7 @@ End-of-life versions will remain available in the npm registry and can still be
 
 We recommend upgrading to Apollo Server 4 as _soon as possible_. Depending on which Apollo Server features you use, the upgrade process might require several changes to your server. The most straightforward upgrade path is to first upgrade from Apollo Server 2 to 3, then once everything works, continue upgrading from Apollo Server 3 to 4.
 
-Three features of Apollo Server 2 have an earlier end-of-life date: **December 31st, 2022**. These three features are:
+Three features of Apollo Server 2 have reached their end-of-life date as of **December 31st, 2022**. These three features are:
 * [Subscriptions](/apollo-server/v2/data/subscriptions/)
 * [File uploads](/apollo-server/v2/data/file-uploads/)
 * [GraphQL Playground](/apollo-server/v2/testing/graphql-playground/).
```

Comment:
The next paragraph could be updated to to be more present-tense.

>  As of 2023, Apollo no longer commits to providing any support for these three features. While you may still file issues about security concerns or significant regressions, if the problems do not exist when a server is initialized with `new ApolloServer({ subscriptions: false, uploads: false, playground: false })`, Apollo may choose not to address them.

or something

Reply:
I like it!